pkgs/tools/security/secp256k1/default.nix

```diff
@@ -16,7 +16,12 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ autoconf automake libtool ];
 
-  configureFlags = [ "--enable-module-recovery" ];
+  configureFlags = [
+    "--enable-experimental"
```

Comment:
Does it make sense to make these flags optional? I mean, when I see `experimental`, I doubt it should be enabled by default.